cfme/rest/gen_data.py

```diff
@@ -334,15 +271,15 @@ def automation_requests_data(vm, requests_collection=False, approve=True):
         "auto_approve": approve
     }
     data = requests_col if requests_collection else automation_requests_col
-    return [data for _ in range(4)]
+    return [data for _ in range(num)]
 
 
 def groups(request, rest_api, role, tenant, num=1):
     data = [{
         "description": "group_description_{}".format(fauxfactory.gen_alphanumeric()),
         "role": {"href": role.href},
         "tenant": {"href": tenant.href}
-    } for index in range(0, num)]
+    } for _ in range(num)]
```

Comment:
OPTIONAL: pep doesn't recommend to use list comprehensions if those cannot fit in one line.